### DISCUSSION TITLE
Rounded labels

### DISCUSSION THREAD

**Discussion by mig-hub** [catppuccin/tmux#385] [Help] [answered]
Sorry, it is an unrelated question, but I've been asking myself how you create these rounded sides on status bar labels. I have tried to copy and paste the characters you have in the code, but I get question marks instead. I am using the Fira code font. Is this because my typeface does not have the proper characters? or is it because there is something else to do for this to work.

Thank you in advance for your answers.

**Comment by kjnsn** [comment]
Are you using a nerd font?

**Reply by mig-hub** [reply]
Fira Code. I suppose it can be considered nerd :-)
But I under the impression these rounded/slanted/chiseled borders are made using a specific font that is loaded with the plugin. At the beginning I thought it was as simple as using these weird Japanese thick brackets. But no...

**Reply by dreknix** [reply, marked as answer]
You can use Fira Code font, but you need a variant with patched glyphs, see https://www.nerdfonts.com/

The already patched Fira Code font can be downloaded via:

https://github.com/ryanoasis/nerd-fonts/releases/download/v3.2.1/FiraCode.zip

**Reply by mig-hub** [reply]
@dreknix Oh I see. This is exactly what I was looking for. Thank you so much !
